modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenResponse.java

```diff
@@ -95,6 +95,7 @@ public class CodegenResponse implements IJsonSchemaValidationProperties {
     private LinkedHashMap<String, CodegenMediaType> content;
     private Map<String, CodegenProperty> requiredVarsMap;
     private String ref;
+    public CodegenProperty returnProperty;
```

Comment:
is this the same as `returnProperty`  already provided?

ref: https://github.com/OpenAPITools/openapi-generator/pull/14379#discussion_r1064636735

Reply:
Yes, but this is on each response, not the operation.

Reply:
👌